### ISSUE TITLE
Feature request: Permission profiles

### ISSUE THREAD

**Issue by xdxita** [CollegeDev/DonkeyGuard#24]
I think that will be great to have user defined profiles where the user can save different permissions configurations(profiles) to easily apply to new or other apps.
I.e. I would have 3 profiles
"trusted apps" (deny: IMPU, IMPI, SIM info, subscriber ID. Allow all others)
"default apps" (deny: IMPU, IMPI, SIM info, subscriber ID, media, contacts. Allow all others)
"untrusted apps" (block all except internet connection and read/write external storage)

Did you get the idea?

**Comment by CollegeDev** [comment]
Yes, I got it. I think this would be a nice idea in combination with a remote-server where every pro user can store his/her profiles and share them among the DonkeyGuard users. What do you think about this?

**Comment by xdxita** [comment]
Sure, it will be great.
BTW can you release a "closed" apk with my translations(pt-BR), so I can fine tune them before public release.
Zum Wohl!

**Comment by Confectrician** [comment]
Remote Server und User Profiles could be a perfekt way to share experiences und settings.

> Zum Wohl!

Perfect @xdxita :+1:

**Comment by xdxita** [comment]
Besides the profile sharing, DG could display the trend for each permission in the apps (i.e. most users set IMPU, IMPI to empty for Facebook app) so those who don't have the knowledge just follow the  crowd.
